engine/execenv/execenv.hpp

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2023 Igor Ostapenko <pm@igoro.pro>
```

Comment:
For consistency with other copyrights and to avoid having to track you down to get your permission later, could you please assign all copyrights to "Kyua Authors", add commit signoff (`git commit --amend -s`), and add yourself to the `AUTHORS` file please?
```suggestion
// Copyright (c) 2024 Kyua Authors
```

Reply:
Fixed.